content/post/2018-06-05_badcomkillsgoodanalytics.md

```diff
@@ -0,0 +1,90 @@
+* Talk about your work
+* Talk about the business
+* Talk about where the data is buried
+
+## Talk as a team
+Each week your company has a number of people digging into data, reading documentation for various programming language modules, and attempting to solve business questions.
+
+Teams focusing on talking together spend less time re-learning the same thing. Their members spend less time being frustrated in solitude, and they enjoy their work because they’re learning more.
+
+Here are some ways I've found to promote good intra-team communication:
+
+###### Crowdsource “I’m stuck”
+Create a safe space to ask questions and discuss shortcomings. It can be a team meeting or a private channel for just the analytics team, but this is crucial to embracing and learning from failure. You’re asking team members to be vulnerable, so set the example in these places by sharing your struggles. The responses should be supportive and in-line with a team committed to growing together.
+
+###### Code Review
+Include a process for documenting and reviewing work in your team’s definition of “done.” The common definition of “done” reminds the team what they’ve agreed to produce. During the code review, the reviewer gets a walk through exactly what, why and how this was done. Additionally, good code review reinforces best practices within the team. Code review for analytics is often substantively different from code review for software engineering because reviewers need to check the business logic and the analytical methods as well as the code.
+
+###### Show and Tell
+Use time during your team meeting to have a team member present something they’ve learned or done. I’d suggest a tempo where a person isn’t asked to present more than once every 4-6 weeks. Be explicit with the goals ahead of time. For example, it might be more valuable for everyone on the team to know what kind of output your Tableau experts can create or why the work was meaningful to a stakeholder, instead of knowing how to create it themselves. A clear purpose helps focus the presentation and the attention of the audience.
+
+## Talk about the business
+I remember bouncing with excitement when pitching my boss an idea for a predictive model assisting another business unit, only to be shown the potential upside of the project could not possibly be worth my time. I had better luck when I found cross-department team members to become collaborators, and as I learned their concerns, I found support for projects that weren’t just interesting to me, but also valuable for the company.
+
+When Analytics teams meet to discuss projects, everyone should understand what drives revenue (and profit), which parts of the business drive the most / least, and what’s the focus/priority for top management. Analytics teams are staffed with curious people. Business context makes their curiosity more likely to end in in projects worth doing.
+
+Here's some techniques to build the team's knowledge about the business:
```

Comment:
Here's -> Here are

Reply:
Actually, there are a number of contractions throughout this post that we'd prefer were expanded. You can check out [the style guide](https://github.com/locallyoptimistic/LocallyOptimistic/blob/master/style_guide.md), but we prefer to have no contractions except don't, can't and doesn't